piaware-ble-connect/led.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+
+# Helper functions to control LEDs on the Pi
+
+import os
+
+def blink(led_num):
+    cmd = f"echo timer | sudo tee /sys/class/leds/{led_num}/trigger 1>/dev/null"
+    os.system(cmd)
+
+def restore_default(led_num):
```

Comment:
Rather than detecting a default based on Pi model, how about we run something in `ExecStartPre` that reads the current setting from sysfs and stores it in a temp file somewhere (under `/var/cache` maybe, or possibly under `/run` if the lifetime of the per-service run dir includes while running `ExecStartPre`/`ExecStopPost`), and then have the `ExecStartPost` restore the recorded setting?